#
# old_revision [8f57d6c713fc0373f6dac132df1f0ac6d6528b3e]
#
# patch "foo"
#  from [83f5a5c359f3dc8317519240e32f1f51f68bc051]
#    to [da39a3ee5e6b4b0d3255bfef95601890afd80709]
#
============================================================
--- foo	83f5a5c359f3dc8317519240e32f1f51f68bc051
+++ foo	da39a3ee5e6b4b0d3255bfef95601890afd80709
@@ -1,2 +0,0 @@
-1
-2
